cmake/Macros.cmake

```diff
@@ -350,7 +350,7 @@ function(sfml_add_external)
             if (NOT include_dir)
                 message(FATAL_ERROR "No path given for include dir ${THIS_INCLUDE}")
             endif()
-            target_include_directories(${target} INTERFACE "$<BUILD_INTERFACE:${include_dir}>")
+            target_include_directories(${target} SYSTEM INTERFACE "$<BUILD_INTERFACE:${include_dir}>")
```

Comment:
Without telling the compiler, that these header are "system" (aka external library) headers, it would scan the thrid-party library headers as well and emit warnings from them.